### ISSUE TITLE
[helm] server helm chart for affinity is wrong

### ISSUE THREAD

**Issue by jmazzitelli** [kiali/kiali#3921]
Looks like we need a simple 1-line bug fixed [here](https://github.com/kiali/helm-charts/blob/v1.33.0/kiali-server/templates/deployment.yaml#L138):

```
      {{- if or (.Values.deployment.affinity.node) (or (.Values.deployment.pod) (.Values.deployment.pod_anti)) }}
```

should be:

```
      {{- if or (.Values.deployment.affinity.node) (or (.Values.deployment.affinity.pod) (.Values.deployment.affinity.pod_anti)) }}
```

(this was reported here: https://github.com/kiali/kiali/issues/3920)

**Comment by rwong2888** [comment]
@jmazzitelli , I didn't see this fixed in 1.33.1

https://github.com/kiali/helm-charts/blob/v1.33.1/kiali-server/templates/deployment.yaml#L138

Any ETA?

**Comment by jmazzitelli** [comment]
@rwong2888 The [PR for that issue](https://github.com/kiali/helm-charts/pull/66) was merged 5 days ago into the master branch, so it is on the next train, scheduled to go into the next release which will be v1.34 whose ETA is next week (Friday, May 7).

It was not part of the emergency patch release v1.33.1 - that was only done to [repair](https://github.com/kiali/kiali-ui/pull/2129 ) a [bug](https://github.com/kiali/kiali/issues/3913) in the UI.
